packages/ckeditor5-ui/src/colorpicker/utils.ts

```diff
@@ -32,6 +38,19 @@ export type ColorPickerConfig = {
 	format?: ColorPickerFormat;
 };
 
+/**
+ * Color converter object that "returns" dedicated function to convert color based on object `key`.
+ */
+
+const convert: any = {
```

Comment:
It should be typed properly, without `any`. In worst case, when you can not type something you could use `unknown`.

Reply:
Yes, agree - this lib should be typed properly :wink:. For now we're using typings from [community types](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/color-convert) .

`unknown` is not working in this case.

If You got any tips that can help - would love to hear them and use them in this particular case.